### ISSUE TITLE
Give citation hint

### ISSUE THREAD

**Issue by KlausIllmayer** [ParthenosWP4/SSK#56]
We should define a way, how the SSK as a whole can be citated. Are there people's name there or do we agree on a more formalized way. There is a dedicated webpage for this (/license) but no text can be found.
From there we should also give hints for the scenarios and the steps, how they can be cited (I guess we can see it as a kind of anthology).

**Comment by charlesriondet** [comment]
There is already a citation template for scenarios, copied in the clipboard when you click on it on a scenario page. Maybe something similar can be done for the steps.
See image:
![image](https://user-images.githubusercontent.com/15829789/64952266-377f1680-d880-11e9-85cb-c24ae7b80a8c.png)

**Comment by KlausIllmayer** [comment]
you are right - my fault, I oversaw it. but what about the general reference, should it be:

The Standardization Survival Kit (SSK), <http://ssk.huma-num.fr>

Is this our official agreed main citation?
What about the date, should we write 2018 on-going?
